animareflection/ui
# Knit

## Description

##### Task links: https://trello.com/c/Q7ZMMCg5/134-replace-yalc-with-knit, https://trello.com/c/eQLpSLzR/158-fix-rendering-issues-in-vite-react-example-app

- Replaced [yalc](https://github.com/wclr/yalc) with [knit](https://github.com/coopbri/knit) for local package publishing
- Fixed `vite-react` example not rendering (used resolutions to pin `@ark-ui/react` to v0.14; error was caused by `ark` (factory function) not being part of older versions, of which v0.9 was being resolved)
 
## Test Steps

- Test familiar local workflow steps with `knit` ([NPM link](https://www.npmjs.com/package/@omnidev/knit)) instead of `yalc`
- Verify `vite-react` example works locally (install deps, link, run)
